docs/api_docs/python/README.md

```diff
@@ -13,18 +13,40 @@ project and is served at
 Either install python packages or, from the build directory:
 
 ```shell
-export PYTHONPATH=$PWD/bindings/python:$PWD/compiler-api/python_package
+source .env && export PYTHONPATH
```

Comment:
Should it be highlighted more clearly that the `.env` file is placed in the build directory?

```suggestion
source ../../../iree-build/.env && export PYTHONPATH
```

Reply:
It's in both the source and build directories: https://github.com/iree-org/iree/blob/41dcee93c7157955d94973addf6770cecf926849/CMakeLists.txt#L993-L1003

(well, the Windows ones are only in the build dir)

Reply:
Still you're probably executing at `docs/api_docs/python` and the `.env` file will be in a different folder if I didn't misread the CMake config.

Reply:
Reworked that section. PTAL.